test/test_unistd.rs

```diff
@@ -191,7 +192,7 @@ fn test_initgroups() {
 }
 
 macro_rules! execve_test_factory(
-    ($test_name:ident, $syscall:ident, $exe: expr $(, $pathname:expr, $flags:expr)*) => (
```

Comment:
Since you're modifying this macro, can you create documentation for it so it's known how it's called?

Reply:
I have no idea how to begin to do this honestly.

Reply:
You modified the macro, is there no documentation you can offer her to help others at least understand your change to it?

Reply:
It's a convoluted test generator macro that became more complex with this change (because there are now separate actions to argument initialization vs coercion for use in exec). I can describe some of the intent but it's not going to make all that much sense without reading the whole macro and seeing what goes where.